.github/workflows/e2e-nightly-35x.yml

```diff
@@ -5,7 +5,6 @@
 
 name: e2e-nightly-35x
 on:
-  workflow_dispatch:  # allow running workflow manually
```

Comment:
I think we might still want this one, for triggering e2e test runs on backports into v0.35.

Reply:
But I think you can do that with the master e2e workflow and just point it to the backport branch. The command that gets executed is the same I believe

Reply:
Ah, right, I guess we are using the same steps either way.